lib/core/application/backend.ts

```diff
@@ -319,10 +313,11 @@ class PluginManager extends ApplicationManager {
    * @param options - Additionnal options
    *    - `name`: Specify plugin name instead of using the class name.
    *    - `manifest`: Manually add a manifest definition (deprecated)
+   *    - `deprecationWarning`: If false, does not display deprecation warnings
```

Comment:
This was intentional otherwise code editor does not displays the options
![image](https://user-images.githubusercontent.com/4447392/100088902-269d3e00-2e51-11eb-9c84-72284acf14c4.png)

![image](https://user-images.githubusercontent.com/4447392/100088927-33ba2d00-2e51-11eb-968f-8d3c9e57acd1.png)

Reply:
...urgh. What the point of writing proper jsdoc comments if they aren't correctly interpreted by code editors... :expressionless: